### PR TITLE
change to self.callback

### PR DESCRIPTION
**Pull request by idohalevi**

### CODE REVIEW

File: lib/logzio-nodejs.js [hunk 1]
```diff
@@ -77,7 +77,12 @@ var jsonToString = exports.jsonToString = function (json) {
 
 LogzioLogger.prototype._defaultCallback = function (err) {
     if (err && !this.supressErrors) {
-        this.internalLogger.log('logzio-logger error: ' + err, err);
+        try {
+            this.internalLogger.log('logzio-logger error: ' + err, err);
+        } catch (e) {
+            console.log('(Using console logger)logzio-nodejs: ' + err, err);
```
Comment (AmirTugi) [inline lib/logzio-nodejs.js:83]: Please use `console.error` for errors.

File: lib/logzio-nodejs.js [hunk 2]
```diff
@@ -260,7 +270,6 @@ LogzioLogger.prototype._send = function (bulk) {
 };
 
 LogzioLogger.prototype._tryToSend = function (options, bulk) {
-    var callback = this.callback;
     var self = this;
 
     function tryAgainIn(sleepTimeMs) {
```
Comment (AmirTugi) [inline lib/logzio-nodejs.js:275]: You can remove the `var self = this` if you use an arrow function `() => {}`

File: lib/logzio-nodejs.js (hunk 1 above)
Comment (AmirTugi) [reply lib/logzio-nodejs.js:83]: Why would this fail, anyway?

**Review by AmirTugi — APPROVED** [review]

**Comment by idohalevi** [comment]
Closing because of #47

**Pull request closed without merging**